### ISSUE TITLE
Get rid of "Ananasz" in Swagger API documentation

### ISSUE THREAD

**Issue by gammaw** [lahmacunradio/arcsi#187]
**Rationale**: "Ananasz" is a legacy term that may confuse us/users reading it in our code/documentation. 

**Example**: https://github.com/lahmacunradio/arcsi/blob/bb3801217803faf58de50a0d9df7ae88e9ad6553/arcsi/static/doc.json#L383

**Misc**: also, not sure if this https://github.com/lahmacunradio/arcsi/tree/master/arcsi/static folder is the standard/most logical location for the Swagger file. What about https://github.com/lahmacunradio/arcsi/tree/master/docs? 

@tuz666 what do you think?

**Comment by tuz666** [comment]
I can agree with the Ananasz renaming, I added this change for my latest PR. Although I don't really know for whom could it be confusing.

About your second suggestion, I'm not sure that moving the Swagger related files out from the arcsi/app folder would be logical. They are used within the [application's context](https://github.com/lahmacunradio/arcsi/blob/master/arcsi/__init__.py#L70), so they are not as separate things as the CI, DB, infra or nginx things.

**Comment by gammaw** [comment]
> Although I don't really know for whom could it be confusing.

Well, we write it for the folks out there, don't we? :)
In Swagger we started using "frontend" for external usage and "archon" for internal, which we can try to stick to. Using another term like "Anananasz" (for "frontend") is confusing.

> Rationale: "Ananasz" is a legacy term that may confuse us/users reading it in our code/documentation.

Fixed via this https://github.com/lahmacunradio/arcsi/pull/185/commits/d63511d09369a16cf19a84c273213739234e0ef3

Thanks @tuz666!
